ccc/grafeas.py

```diff
@@ -1,24 +1,13 @@
 import logging
 import urllib.parse
 
-try:
-    from grafeas.grafeas_v1.gapic.transports.grafeas_grpc_transport import GrafeasGrpcTransport
```

Comment:
do we not need to do this (again) in a two-step change (first, upgrade dependencies + release, then remove backward-compatibility coding, and release again)?